apps/web/pages/auth/login.tsx

```diff
@@ -256,12 +256,10 @@ inferSSRProps<typeof getServerSideProps> & WithNonceProps<{}>) {
                     CustomStartIcon={<GoogleIcon />}
                     onClick={async (e) => {
                       e.preventDefault();
-                      const res = await signIn("google", {
+                      setLastUsed("google");
+                      await signIn("google", {
```

Comment:
Are these changes accidental?  If not we should handle it in a separate PR

Reply:
Not accidental, small improvement for my merged pr. No needed to create a separate pr for this small changes

Reply:
@anikdhabal This isn't a good practice to have unrelated changes in the PR. Two major reasons
1. Unrelated changes might delay the original fix(if reviewer have concern with the unrelated changes code quality/stability)
2. More importantly, the PR can be reverted and that reverts the unrelated change in the PR which people won't be aware of.


So, could you please remove these changes from this PR.

Reply:
Done